backend/src/services/background/old-instances-auto-deleting/old.insnaces.bj.service.ts

```diff
@@ -0,0 +1,37 @@
+import { instance as InstanceRep } from '~/data/repositories/repositories';
```

Comment:
lets name the service as `background-job`. it will be responsible for all background jobs in our app

Reply:
I think it is a bad idea